app/src/main/java/org/jellyfin/androidtv/util/apiclient/PlaybackHelper.java

```diff
@@ -34,9 +35,15 @@
 import static org.koin.java.KoinJavaComponent.get;
 
 public class PlaybackHelper {
-    private static final int limit = 150; // guard against too many items - account for episodes shorter than 5 min
+    private static final int ITEM_QUERY_LIMIT = 150; // limit the number of items retrieved for playback
 
     public static void getItemsToPlay(final BaseItemDto mainItem, boolean allowIntros, final boolean shuffle, final Response<List<BaseItemDto>> outerResponse) {
+        final UserDto currentUser = TvApp.getApplication().getCurrentUser();
+        if (currentUser == null) {
+            Timber.e("Unable to get items to play; current user is null");
```

Comment:
Shoudln't `outerResponse` be called here so the code doesn't keep waiting for a response?

Reply:
Yeap good call. Fixed now.